compiler/src/org.graalvm.compiler.truffle.options/src/org/graalvm/compiler/truffle/options/PolyglotCompilerOptions.java

```diff
@@ -391,6 +391,9 @@ private String indent(int nameLength) {
     @Option(help = "Maximum depth for recursive inlining.", category = OptionCategory.EXPERT)
     public static final OptionKey<Integer> InliningRecursionDepth = new OptionKey<>(2);
 
+    @Option(help = "Exclude from being inlined for ','-separated list of methods.", category = OptionCategory.INTERNAL)
```

Comment:
Help message is a bit confusing, maybe `A  ','-separated list of method names that should be never be inlined`